lametro/templatetags/lametro_extras.py

```diff
@@ -356,11 +356,12 @@ def bill_status_from_last_action(description):
 
 
 @register.inclusion_tag("snippets/fiscal_year_calendars.html", takes_context=True)
-def fiscal_year_calendars(context):
+def fiscal_year_calendars(context, stacked=True):
```

Comment:
This is to match the styles for the calendars that are currently on the live site. The index has the calendars stacked on top of each other by default, while the events page has them side-by-side at larger screen sizes.